akka-stream/src/main/scala/akka/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -99,7 +100,7 @@ private[akka] case class ActorMaterializerImpl(
             assignPort(stage.outPort, processor)
             mat
 
-          case tls: TlsModule ⇒
+          case tls: TlsModule ⇒ // TODO isn't this a bit of a hack?
```

Comment:
Ok, in that case, I love TODOs/FIXMEs so that the casual reader thinks this is permanent :)